createScreenshot.sh

```diff
@@ -6,6 +6,6 @@ then
     exit
 fi
 
-bash fetchScreen.sh $1 $2.png
+bash screenshotScript/fetchScreen.sh $1 $2.png
```

Comment:
Maybe call it `scripts`, then we could also migrate our Python stuff there.

Reply:
Hmm.. how about `Tools`?

Reply:
I think we should move all the script tools(`*.py`,`*.sh`...etc).
```
Open-smartwatch/
└── scripts/
        ├── Python/
        └── Shell/
```

Reply:
`scripts` get's my vote 👍

Reply:
Fair